### PR TITLE
chore: prepare for v2.11.0 release

### PR DESCRIPTION
**Pull request by dbolduc**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762)

### CODE REVIEW

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @dbolduc)

___
*[`CHANGELOG.md` line 133 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762#-NWrb7wDEERHp7HxxW7r:-NWrb7wDEERHp7HxxW7s:bfymv0p) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/bae56096f0beb989414a6c2d3f262762095a0706/CHANGELOG.md#L133)):*
> ```Markdown
> The following experimental libraries are now available:
> 
> - [Compute Engine API](/google/cloud/compute/README.md)
> ```

Let's wait on announcing this until next month as only one service is currently present.



_Code quote:_
```Markdown
The following experimental libraries are now available:

- [Compute Engine API](/google/cloud/compute/README.md)
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -128,11 +128,53 @@ the APIs in these libraries are stable, and are ready for production use.
...
+
+### [Spanner](/google/cloud/spanner/README.md)
+
+- docs(spanner): some documentation tweaks ([#11641](https://github.com/googleapis/google-cloud-cpp/pull/11641))
+- feat(spanner): tests and samples for drop-database protection ([#11637](https://github.com/googleapis/google-cloud-cpp/pull/11637))
+- feat(spanner): make `ResultSourceInterface` public ([#11636](https://github.com/googleapis/google-cloud-cpp/pull/11636))
+
+### [Storage](/google/cloud/storage/README.md)
+
+- docs(storage): add GCS+gRPC release notes ([#11760](https://github.com/googleapis/google-cloud-cpp/pull/11760))
```
Comment (coryan) [inline CHANGELOG.md:153]: I think you are missing https://github.com/googleapis/google-cloud-cpp/pull/11754

**Review by dbolduc — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @coryan and @scotthart)

___
*[`CHANGELOG.md` line 133 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762#-NWrb7wDEERHp7HxxW7r:-NWrf-Fx-WoXPduw2HUI:bombmv2) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/bae56096f0beb989414a6c2d3f262762095a0706/CHANGELOG.md#L133)):*
<details><summary><i>Previously, scotthart (Scott Hart) wrote…</i></summary><blockquote>

Let's wait on announcing this until next month as only one service is currently present.
</blockquote></details>

Good call. Done.
___
*[`CHANGELOG.md` line 153 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762#gh-1213354523:-NWrf0nV8ScsSRA_r62-:bfvbwa3) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/bae56096f0beb989414a6c2d3f262762095a0706/CHANGELOG.md#L153)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

I think you are missing https://github.com/googleapis/google-cloud-cpp/pull/11754 
</blockquote></details>

Thanks. I forgot to rebase.

**Review by scotthart — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11762)* status: all files reviewed, 2 unresolved discussions (waiting on @coryan)

**Review by coryan — APPROVED** [review]

**Pull request merged**